test/units/modules/network/edgeos/test_edgeos_command.py

```diff
@@ -33,4 +33,76 @@ class TestEdgeosCommandModule(TestEdgeosModule):
 
     def setUp(self):
         super(TestEdgeosCommandModule, self).setUp()
-        self.mock_run_commands = patch()
+        self.mock_run_commands = patch('ansible.modules.network.edgeos.edgeos_command.run_commands')
+        self.run_commands = self.mock_run_commands.start()
+
+    def tearDown(self):
+        super(TestEdgeosCommandModule, self).tearDown()
+        self.mock_run_commands.stop()
+
+    def load_fixtures(self, commands=None):
+        def load_from_file(*args, **kwargs):
+            module, commands = args
+            output = list()
+
+            for item in commands:
+                print(type(item))
+                try:
+                    obj = json.loads(item)
+                    command = obj['command']
+                except ValueError:
+                    command = item
```

Comment:
Below change will fix the ` TypeError: the JSON object must be str, not 'dict'` error
```
                try:
                    obj = json.loads(item)
                    command = obj['command']
                except (ValueError, TypeError):
                    command = item['command']
```

Reply:
Thanks! That worked well.